### PR TITLE
add missing parameters of the commit API to the docs

### PR DESCRIPTION
**Pull request by mountkin**
Signed-off-by: Shijiang Wei mountkin@gmail.com

### CODE REVIEW

**Comment by thaJeztah** [comment]
LGTM, thanks!

**Comment by mountkin** [comment]
ping @moxiegirl :smile_cat:

**Comment by thaJeztah** [comment]
@moxiegirl is quite busy at the moment, so

ping @fredlf @jamtur01 @SvenDowideit PTAL

File: docs/reference/api/docker_remote_api_v1.20.md [hunk 1]
```diff
@@ -1892,6 +1892,8 @@ Query Parameters:
 -   **comment** – commit message
 -   **author** – author (e.g., "John Hannibal Smith
     <[hannibal@a-team.com](mailto:hannibal%40a-team.com)>")
+-   **pause** – 1/True/true or 0/False/false, whether to pause the container before commiting
```
Comment (jamtur01) [inline docs/reference/api/docker_remote_api_v1.20.md]: committing

File: docs/reference/api/docker_remote_api_v1.20.md [hunk 2]
```diff
@@ -1892,6 +1892,8 @@ Query Parameters:
...
+-   **changes** – Dockerfile instructions to apply while commiting
```
Comment (jamtur01) [inline docs/reference/api/docker_remote_api_v1.20.md]: committing

File: docs/reference/api/docker_remote_api_v1.21.md [hunk 3]
```diff
@@ -1987,6 +1987,8 @@ Query Parameters:
 -   **comment** – commit message
 -   **author** – author (e.g., "John Hannibal Smith
     <[hannibal@a-team.com](mailto:hannibal%40a-team.com)>")
+-   **pause** – 1/True/true or 0/False/false, whether to pause the container before commiting
+-   **changes** – Dockerfile instructions to apply while commiting
```
Comment (jamtur01) [inline docs/reference/api/docker_remote_api_v1.21.md]: Spelling - committing

**Comment by jamtur01** [comment]
Needs spelling fixed.

**Comment by mountkin** [comment]
Fixed.  Thanks a lot @jamtur01

File: docs/reference/api/docker_remote_api_v1.20.md (hunk 1 above)
Comment (thaJeztah) [inline docs/reference/api/docker_remote_api_v1.20.md]: ow, good catch, thx

**Comment by thaJeztah** [comment]
thanks for fixing @mountkin, think it's good to go, merging!

**Pull request merged**